primary/src/routes/scouting.ts

```diff
@@ -70,25 +76,37 @@ router.get('/match*', wrap(async (req, res) => {
 		{allowCache: true}
 	);
 	
-	
 	const images = await uploadHelper.findTeamImages(org_key, eventYear, teamKey);
-	const team: Team = await utilities.findOne('teams', {key: teamKey}, {}, {allowCache: true});
+	
+	let team: Team;
+	
+	// Check if team has a letter at the end, e.g. frc102B
+	let letterTeamMatch = /(frc\d+)[a-zA-Z]$/g.exec(teamKey);
+	// if so, just find the info for the team without the letter on the end (found w/ a capturing group1)
+	if (letterTeamMatch) {
+		team = await utilities.findOne('teams', {key: letterTeamMatch[1]}, {}, {allowCache: true});
+	}
+	else {
+		team = await utilities.findOne('teams', {key: teamKey}, {}, {allowCache: true});
+	}
```

Comment:
This looks like it should work, but I think this could potentially cause other errors to happen in stats pages- maybe we should abstract this code to a general getTeam function that can compensate for letter team keys.